modules/running-distributed-data-science-workloads-from-ds-pipelines.adoc

```diff
@@ -38,11 +38,11 @@ ifndef::upstream[]
 +
 [NOTE]
 ====
-If you do not create a default local queue, you must specify a local queue in each pipeline.
+If your cluster administrator does not define a default local queue, you must specify a local queue in each pipeline.
 ====
 endif::[]
 ifdef::upstream[]
-* Optional: You have defined a _default_ local queue for the Ray cluster by creating a `LocalQueue` resource and adding the following annotation to the configuration details for that `LocalQueue` resource, as described in link:{odhdocshome}/working-with-distributed-workloads/#configuring-quota-management-for-distributed-workloads_distributed-workloads[Configuring quota management for distributed workloads]:
+* Optional: Your cluster administrator has defined a _default_ local queue for the Ray cluster by creating a `LocalQueue` resource and adding the following annotation to the configuration details for that `LocalQueue` resource, as described in link:{odhdocshome}/working-with-distributed-workloads/#configuring-quota-management-for-distributed-workloads_distributed-workloads[Configuring quota management for distributed workloads]:
```

Comment:
Same comment as above: "and has added".

Reply:
As explained above, the existing text is correct.